### PR TITLE
Network  Automation and Automation Tools

### PR DESCRIPTION
**Pull request by raboba2re**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by raboba2re** [comment]
This fixes #2760

File: content/articles/network-automation/index.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+#### Chef 
+![Chef](/network-automation/chef.png)
+[source](https://en.wikipedia.org/wiki/Chef_(software))
+ Just like Puppet, Chef is written in Ruby programming language. Users describe how it manages applications and utilities using written **recipes** which are then grouped in a **cookbook** for easy management.  The recipes describe which packages should be installed and which services should be running [Wikipedia](https://en.wikipedia.org/wiki/Chef_(software)), to accelerate application delivery, DeOPs collaboration, and network uptime. 
+
+Chef is an agent-based configuration management tool that uses “chef master to manage orchestration and automation of network tacks. 
+
+### Summary 
+In this article, the reader have learned:
```
Comment (mbaabuones) [inline content/articles/network-automation/index.md:110]:
```suggestion
In summary: 
- We have gained an overview of network automation 
- We have learned what APIs are
- We have gone through various types of web service APIs
- We have understood what network management tools are 
- We have learned four network management tools: Ansible, Puppet, Saltstack, and Chef.

**Review by mbaabuones — CHANGES_REQUESTED** [review]
@raboba2re Great article! The article is informative and well structured. I have noted some typos. Please go through the article keenly and correct all the grammar errors. In your subsequent PRs, ensure you proofread your article before submitting it. If possible, use a grammar review tool to check whether there are grammar errors (eg. grammarly). Please work on the suggested changes so that we can move to the next step.

**Comment by raboba2re** [comment]
@mbaabuones I have effected  the corrections you requested.
 Sorry for those typos. I opened the PR using the wrong draft.

**Comment by hectorkambow** [comment]
plagiarism check done

**Review by mbaabuones — APPROVED** [review]

**Review by mbaabuones — APPROVED** [review]

**Comment by WanjaMIKE** [comment]
@mbaabuones please check some of the minor edits I have made in the final review. Thanks

**Comment by WanjaMIKE** [comment]
Merged #3052

**Pull request merged**